File: app/src/main/java/com/example/mediasearchdemo/MainActivity.java
package com.example.mediasearchdemo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import java.util.List;

import kotlin.Unit;
import kotlin.jvm.functions.Function1;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        String[] strings = new String[]{MediaStoreUtils.WORD};

        MediaStoreUtils.INSTANCE.loadMedia(this, strings, new Function1<List<? extends MediaFolderBean>, Unit>() {
            @Override
            public Unit invoke(List<? extends MediaFolderBean> mediaFolderBeans) {
                //查询出所有word文件
                List<? extends MediaFolderBean> a = mediaFolderBeans;


                return null;
            }
        });
    }
}

File: app/src/main/java/com/example/mediasearchdemo/MainActivity.kt
package com.example.mediasearchdemo

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.example.mediasearchdemo.fileSelector.LoadManagerUtils
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_search.setOnClickListener {
            val intent = Intent(this, FileSelectorActivity::class.java)
            intent.putExtra("type",0)
            startActivity(intent)
        }

        btn_search2.setOnClickListener {
            val intent = Intent(this, FileSelectorActivity::class.java)
            intent.putExtra("type",1)
            startActivity(intent)
        }
    }
}
